specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```
@@ -0,0 +1,6113 @@
+        "operationId": "EnrollmentGroups_Get",
+        "summary": "Get an enrollment group by ID",
+        "description": "Get details about an enrollment group by ID.",
+        "x-ms-examples": {
+          "Get an enrollment group by ID": {
+            "$ref": "./examples/enrollmentgroups_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/EnrollmentGroupId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EnrollmentGroup"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "EnrollmentGroups_Create",
+        "summary": "Create an enrollment group",
+        "description": "Create an enrollment group.",
+        "x-ms-examples": {
+          "Get an enrollment group": {
+            "$ref": "./examples/enrollmentgroups_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/EnrollmentGroupId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Enrollment group body.",
+            "schema": {
+              "$ref": "#/definitions/EnrollmentGroup"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EnrollmentGroup"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "EnrollmentGroups_Update",
+        "summary": "Update an enrollment group",
+        "description": "Update an enrollment group.",
+        "x-ms-examples": {
+          "Update an enrollment group by ID": {
+            "$ref": "./examples/enrollmentgroups_update.json"
+          }
+        },
+        "consumes": [
+          "application/json",
+          "application/merge-patch+json"
```

Patch operation should only accept `application/merge-patch+json`.
```suggestion
          "application/merge-patch+json"
```

---

Fixed